src/factories/fardetectors/FarDetectorTrackerCluster_factoryT.h

```diff
@@ -0,0 +1,77 @@
+// Created by Simon Gardner
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
```

Comment:
This directory includes LGPL3 factories. They were written from scratch as multifactories, not based on the older factories elsewhere. If you used other factories in this directory as a model, the LGPL3 license applies and should be in the header.